### PR TITLE
Make ArchiBot tell the truth

### PR DESCRIPTION
**Pull request by Abrynos**

### CODE REVIEW

**Comment by Thielak** [comment]
@JustArchi could probably shed some light on the matter

File: Brain/ArchiBot/Memorized/_learned.siml [hunk 1]
```diff
@@ -140,11 +140,11 @@
       </Pattern>
       <Response>
         <Phrase>
-          <Item>abrynos is a weeb</Item>
-          <Item>You told me that abrynos is a weeb</Item>
-          <Item>I rember that abrynos is a weeb</Item>
+          <Item>abrynos is a shitty programmer with even shittier humour</Item>
+          <Item>You told me that abrynos is a shitty programmer with even shittier humour</Item>
+          <Item>I rember that abrynos is a shitty programmer with even shittier humour</Item>
```
Comment (JustArchi) [inline Brain/ArchiBot/Memorized/_learned.siml:145]: I could spend my entire life searching for an answer and never find it. I could spend a lifetime in labs, write my PhD thesis on the search for it; I could go deep sea diving, I could climb the highest mountains, I could go on expeditions to both poles, I could send deep-space probes, I could go beyond the edge of the observable universe, I could enter a black hole trying to find it, I could be looking the whole time from the big bang to the heat death of the universe, and I would still never find the definite proof that abrynos is not a weeb.

**Comment by JustArchi** [comment]
Hey @Thielak,

Hope you are save and well up during this time.

I have sort of an unusual question, looking forward to your help:

I want to get in touch with Kaleith in order to hear his thoughts on the code review I wrote a few minutes ago (hoping this review should be of interest to him). So all I want is some minutes of his scarce time and that he hears me.

Contacting him should be pretty complicated without any personal relationship. Also, as I am from Europe, I can not just randomly meet him at a conference or another event.

So what I am looking for is someone, who knows Kaleith personally (e.g. from working at 2hu) and can ask him to specifically read my review above.

Thank you - really appreciate your help!

**Comment by JustArchi** [comment]
Actually, what a day. My mom was nice enough to go grab our family some Burger King. They got the order wrong (it was an expensive order, and it was really really wrong) , okay, it happens. No one answers the phone call. Fine, they're busy. My mom goes back, and she had the manager blame her, she had a worker photograph her, and the window cashier crumpled the receipt and threw it at her. I'm so enraged. There's nothing we can even do. It was the BK on Rice and Larpenteur East, and don't give me any defense for the employees, we've all had shitty jobs. The actions of the staff were reprehensible. I posted a picture of my car because this experience was a wreck. I might as well merge this.

File: Brain/ArchiBot/Memorized/_learned.siml (hunk 1 above)
Comment (Thielak) [reply Brain/ArchiBot/Memorized/_learned.siml:145]: I was sharing a passionate kiss with my romantic interest, when my cellular phone alerted me that someone wished to converse with me. The man on the phone inquired as to what I was engaging in with his daughter. Assuming this man was my romantic interest's father, I explained the conversation I had to her. She informed me that her father was deceased. That is when I thought to myself "If her father is dead, then who was conversing with me on the cellular phone?" And then a human skeletal structure appeared.

**Comment by Thielak** [comment]
I can't believe you did that. May I speak to your manager?

**Pull request merged**